Author: aurel32
Status: ugly, don't merge! We need the new dpkg to make a clean patch.

diff -u polypaudio-0.7+20050805/debian/control polypaudio-0.7+20050805/debian/control
--- polypaudio-0.7+20050805/debian/control
+++ polypaudio-0.7+20050805/debian/control
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Otavio Salvador <otavio@debian.org>
 Uploaders: Martin Pitt <mpitt@debian.org>
-Build-Depends: cdbs (>= 0.4.23-1.1), build-essential, debhelper (>= 4.1.0), patchutils (>= 0.2.25), m4, libwrap0-dev, libsamplerate0-dev (>= 0.1.0), libsndfile1-dev (>= 1.0.0), libasound2-dev (>= 1.0.0), libglib2.0-dev (>= 2.4.0), libxt-dev, libltdl3-dev
+Build-Depends: cdbs (>= 0.4.23-1.1), build-essential, debhelper (>= 4.1.0), patchutils (>= 0.2.25), m4, libwrap0-dev, libsamplerate0-dev (>= 0.1.0), libsndfile1-dev (>= 1.0.0), libasound2-dev (>= 1.0.0) [!kfreebsd-i386 !hurd-i386], libglib2.0-dev (>= 2.4.0), libxt-dev, libltdl3-dev
 Standards-Version: 3.6.2
 
 Package: polypaudio
@@ -30,7 +30,7 @@
 
 Package: polypaudio-alsa
 Section: sound
-Architecture: any
+Architecture: i386 ia64 alpha amd64 arm hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, polypaudio
 Replaces: polypaudio
 Description: ALSA modules for the polypaudio sound server
